modules/imgproc/src/filter.simd.hpp

```diff
@@ -1806,28 +1803,28 @@ struct SymmRowSmallVec_32f
                 {
 #if CV_FMA3 || CV_AVX2
```

Comment:
Is CV_FMA3 defined for RVV? If it's not relevant, I propose to not touch this block.

Reply:
CV_FMA3 is not defined for RVV. But I think that in order to maintain the consistency of the universal intrinsic API, we should modify the old code as much as possible, as long as it can ensure correctness and performance.

So although these code blocks has nothing to do with the RVV backend, it should be modified as well.

Reply:
@asmorkalov , the end goal is to remove unsupported constructions (nlanes, operators) from all intriniscs, so that it will not be possible to write non-scalable code.